openstack/telemetry/v2/meters/fixtures.go

```diff
@@ -0,0 +1,206 @@
+//// +build fixtures
```

Comment:
@DSpeichert I thought I had done a PR on your branch on this line, but it seems I imagined it, can't find even find it on my own repo any more... The "double-comment" causes this to always be built, and since it imports `testing`, anything built against it will be identified as a test build (and `flag` will add a bunch of testing flags, which is how I noticed)